options/options.go

```diff
@@ -45,21 +44,9 @@ func WithNamespace(namespace string) SetOptions {
 	}
 }
 
-// WithDestroy returns a closure that sets an option so that instead of creating a
-// tombstone, the data is permanently deleted from the database, meaning that a
-// subsequent Put will restart the versioning. Use Destroy with care in a distributed
-// system, if you destroy a key in an anti-entropy environment it will simply be
-// repaired by the system to the latest version before the Delete.
-func WithDestroy() SetOptions {
-	return func(cfg *Options) error {
-		cfg.Destroy = true
-		return nil
-	}
-}
```

Comment:
In #15 we decided that we wanted mirror methods for replica operations and user operations so we should have a `honu.Destroy` method.